R/annotate.R

```diff
@@ -0,0 +1,52 @@
+# Annotate Graphs (should this go into gGtools.R?)
+#
+#' Produce annotated gene graph where vertices have scores
+#' attributed to proper genes.
+#'
```

Comment:
"proper genes"? No need to check for the semantics of the data in this function - just match IDs...